.release-please-manifest.json

```
@@ -1,3 +1,4 @@
 {
-  ".": "1.12.0"
+  ".": "1.12.0",
+  "operator/.": "0.1.0"
```

I think we just want `operator` here

```suggestion
  "operator": "0.1.0"
```